app/screens/home/search/initial/modifiers/index.tsx

```diff
@@ -4,99 +4,98 @@
 import React, {type Dispatch, type RefObject, type SetStateAction, useCallback, useEffect, useMemo, useRef, useState} from 'react';
 import {type IntlShape, useIntl} from 'react-intl';
 import {View} from 'react-native';
-import Animated, {useSharedValue, useAnimatedStyle, withTiming} from 'react-native-reanimated';
+import Animated, {type SharedValue, useSharedValue, useAnimatedStyle, withTiming} from 'react-native-reanimated';
 
 import FormattedText from '@components/formatted_text';
 import {useTheme} from '@context/theme';
-import TeamPickerIcon from '@screens/home/search/team_picker_icon';
+import TeamPicker from '@screens/home/search/team_picker';
 import {makeStyleSheetFromTheme} from '@utils/theme';
 import {typography} from '@utils/typography';
 
+import {ALL_TEAMS_ID} from '../..';
+
 import Modifier, {type ModifierItem} from './modifier';
 import ShowMoreButton from './show_more';
 
 import type {SearchRef} from '@components/search';
 import type TeamModel from '@typings/database/models/servers/team';
 
 const MODIFIER_LABEL_HEIGHT = 48;
-const TEAM_PICKER_ICON_SIZE = 32;
 const NUM_ITEMS_BEFORE_EXPAND = 4;
 
 const getStyleFromTheme = makeStyleSheetFromTheme((theme) => {
     return {
-        titleContainer: {
+        header: {
             alignItems: 'center',
             flexDirection: 'row',
             marginTop: 20,
-            marginRight: 18,
+            marginHorizontal: 18,
         },
-        title: {
+        titleContainer: {
             flex: 1,
+        },
+        title: {
             alignItems: 'center',
-            paddingLeft: 18,
             color: theme.centerChannelColor,
             ...typography('Heading', 300, 'SemiBold'),
         },
+        teamPickerContainer: {
+            flex: 1,
+            alignItems: 'flex-end',
+        },
     };
 });
 
-const getModifiersSectionsData = (intl: IntlShape): ModifierItem[] => {
+const getModifiersSectionsData = (intl: IntlShape, teamId: string): ModifierItem[] => {
     const formatMessage = intl.formatMessage;
-    const sectionsData = [
-        {
+
+    const sectionsData = [];
+    if (teamId !== ALL_TEAMS_ID) {
+        sectionsData.push({
             term: 'From:',
             testID: 'search.modifier.from',
             description: formatMessage({id: 'mobile.search.modifier.from', defaultMessage: ' a specific user'}),
-        }, {
+        });
+
+        sectionsData.push({
```

Comment:
you could push the two items at the same time, any reason to change it?

Reply:
I didn't know push could take multiple args :) Thanks!